src/main/scala/controllers/ApiHelpController.scala

```diff
@@ -61,11 +61,24 @@ class ErrorResponse(@XmlElement var code: Int, @XmlElement var message: String)
   def setMessage(message: String) = this.message = message
 }
 
+/**
+ * For Java8, add support for a `aString.is_blank` function.
+ * Cannot be called `isBlank` since that conflicts with Java 11's `java.lang.String.isBlank`.
+ * Can be removed in favor of `java.lang.String.isBlank` when updating to Java 11.
+ */
+object Java8Utils {
+  implicit class StringExtension(val s: String) extends AnyVal {
+    def is_blank: Boolean = s.trim == ""
+  }
+}
+
 class ApiHelpController @Inject() (components: ControllerComponents, val swaggerPlugin: SwaggerPlugin)
   extends AbstractController(components) with SwaggerBaseApiController {
 
+  import Java8Utils.StringExtension
+
   def getResources = Action { implicit request =>
-    val host: String = swaggerPlugin.config.host
+    val host = if (swaggerPlugin.config.host.is_blank) request.host else swaggerPlugin.config.host
```

Comment:
I agree that `localhost:9000` is a poor default and your PR is a nice improvement.

But can't we leave the host blank here? Because Swagger already derives the host from the current URL if the host is missing from `swagger.json`.

Reply:
Indeed, you're right. Setting the `swagger.api.host` property to `""` results in a base-url used on the Swagger UI without a host, only `/`. And therefore will perform a request to the actual host/port.

So that makes this PR not really required anymore to fix the issue.
Or it could be reduced to removing the default value for `swagger.api.host` and just make it `""` so not everyone has to do this in their project.
Nevertheless, I think it would be nice to display the actual host/port instead of just `/`.

Reply:
Could you update this PR to just remove the `localhost:9000` default? We can discuss replacing the host as another issue.

Reply:
Reverted changes in `ApiHelpController`